### ISSUE TITLE
update argument is not resulting updating the media information

### ISSUE THREAD

**Issue by laszlojanosek** [deadc0de6/catcli#15]
Not sure if this is intentional, or I miss a parameter. If neither of these then I think it is a feature request for your consideration.

Update argument is updating the content of the folder but the media information (size, free space, date, etc.) remains as it was captured during the original index.

Example:
1. Index a folder to a catalogue file.

```
   catcli index --meta="whatever" --catalog=whatever.catalog 191 /media/user/191
```

2. list content of the media within the catalog shows (<name> is 191)

```
   catcli ls --catalog=whatever.catalog 191
   storage: 191                  (whatever)
     nbfiles:1     ,totsize:204.8G ,free:33.2% ,du:91.1G/274.0G  ,date: 2021-08-26 15:52:46
```

3. Changes in the folder
4. Update the catalog

```
   catcli update --catalog=whatever.catalog 191 /media/user/191
```

5. "catcli ls --catalog=whatever.catalog 191" still shows the size information captured during the original index (as in example step 2.)

As a workaround, I remove the <name> from the catalog and index it again. It would be nice if the media information would be updated as well as the date. I use this parameter in later scripts to ease finding available storage space in backup media.

BTW, I still love this piece of software. ;-)

**Comment by deadc0de6** [comment]
Thanks for submitting this.
You are right, `update` should definitely update the media information. I'll look into it!

**Comment by deadc0de6** [comment]
I fixed it and released version [0.7.4](https://pypi.org/project/catcli/). Thanks for your help!
